diff --git "a/platform/ext/target/stm/b_u585i_iot02a/accelerator/mbedtls_accelerator_config.h" "b/platform/ext/target/stm/b_u585i_iot02a/accelerator/mbedtls_accelerator_config.h"
index 97c589fb..8dac6cc3 100644
--- "a/platform/ext/target/stm/b_u585i_iot02a/accelerator/mbedtls_accelerator_config.h"
+++ "b/platform/ext/target/stm/b_u585i_iot02a/accelerator/mbedtls_accelerator_config.h"
@@ -21,45 +21,45 @@ extern "C" {
 #define MBEDTLS_ENTROPY_C
 #define MBEDTLS_ENTROPY_HARDWARE_ALT
 
-#ifdef MBEDTLS_SHA1_C
-#define MBEDTLS_SHA1_ALT
-#endif /* MBEDTLS_SHA1_C */
+// #ifdef MBEDTLS_SHA1_C
+// #define MBEDTLS_SHA1_ALT
+// #endif /* MBEDTLS_SHA1_C */
 
-#ifdef MBEDTLS_SHA256_C
-#define MBEDTLS_SHA256_ALT
-#endif /* MBEDTLS_SHA256_C */
+// #ifdef MBEDTLS_SHA256_C
+// #define MBEDTLS_SHA256_ALT
+// #endif /* MBEDTLS_SHA256_C */
 
-#undef MBEDTLS_AES_SETKEY_DEC_ALT
-#undef MBEDTLS_AES_DECRYPT_ALT
+// #undef MBEDTLS_AES_SETKEY_DEC_ALT
+// #undef MBEDTLS_AES_DECRYPT_ALT
 
-#ifdef MBEDTLS_RSA_C
-#define MBEDTLS_RSA_ALT
-#endif /* MBEDTLS_RSA_C */
+// #ifdef MBEDTLS_RSA_C
+// #define MBEDTLS_RSA_ALT
+// #endif /* MBEDTLS_RSA_C */
 
-#if defined(MBEDTLS_ECP_C)
-#define MBEDTLS_ECP_ALT
-#undef MBEDTLS_ECP_NIST_OPTIM
-/*#define MBEDTLS_MD5_ALT*/
-#endif /* MBEDTLS_ECP_C && MBEDTLS_MD_C */
+// #if defined(MBEDTLS_ECP_C)
+// #define MBEDTLS_ECP_ALT
+// #undef MBEDTLS_ECP_NIST_OPTIM
+// /*#define MBEDTLS_MD5_ALT*/
+// #endif /* MBEDTLS_ECP_C && MBEDTLS_MD_C */
 
-#ifdef MBEDTLS_CCM_C
-#define MBEDTLS_CCM_ALT
-#endif
-#ifdef MBEDTLS_AES_C
-#define MBEDTLS_AES_ALT
-#endif
-#ifdef MBEDTLS_GCM_C
-#define MBEDTLS_GCM_ALT
-#endif
-#ifdef MBEDTLS_ECDSA_C
-#define MBEDTLS_ECDSA_VERIFY_ALT
-#define MBEDTLS_ECDSA_SIGN_ALT
-#endif
+// #ifdef MBEDTLS_CCM_C
+// #define MBEDTLS_CCM_ALT
+// #endif
+// #ifdef MBEDTLS_AES_C
+// #define MBEDTLS_AES_ALT
+// #endif
+// #ifdef MBEDTLS_GCM_C
+// #define MBEDTLS_GCM_ALT
+// #endif
+// #ifdef MBEDTLS_ECDSA_C
+// #define MBEDTLS_ECDSA_VERIFY_ALT
+// #define MBEDTLS_ECDSA_SIGN_ALT
+// #endif
 /* specific Define for platform hardware accelerator */
-#define GENERATOR_HW_PKA_EXTENDED_API
-#define GENERATOR_HW_CRYPTO_DPA_SUPPORTED
-#define HW_CRYPTO_DPA_AES
-#define HW_CRYPTO_DPA_GCM
+// #define GENERATOR_HW_PKA_EXTENDED_API
+// #define GENERATOR_HW_CRYPTO_DPA_SUPPORTED
+// #define HW_CRYPTO_DPA_AES
+// #define HW_CRYPTO_DPA_GCM
 #ifdef __cplusplus
 }
 #endif /* __cplusplus */
