diff --git "a/platform/ext/target/stm/b_u585i_iot02a/include/flash_layout.h" "b/platform/ext/target/stm/b_u585i_iot02a/include/flash_layout.h"
index 7f817c65..00d4cbec 100644
--- "a/platform/ext/target/stm/b_u585i_iot02a/include/flash_layout.h"
+++ "b/platform/ext/target/stm/b_u585i_iot02a/include/flash_layout.h"
@@ -79,7 +79,7 @@
 
 /* area for BL2 code protected by hdp */
 #define FLASH_AREA_BL2_OFFSET           (FLASH_AREA_PERSO_OFFSET+FLASH_AREA_PERSO_SIZE )
-#define FLASH_AREA_BL2_SIZE             (0x12000)
+#define FLASH_AREA_BL2_SIZE             (0x18000)
 /* HDP area end at this address */
 #define FLASH_BL2_HDP_END               (FLASH_AREA_BL2_OFFSET+FLASH_AREA_BL2_SIZE-1)
 /* area for BL2 code not protected by hdp */
@@ -120,7 +120,7 @@
 #endif /*  (FLASH_ITS_AREA_OFFSET % FLASH_AREA_IMAGE_SECTOR_SIZE) != 0 */
 
 #define FLASH_S_PARTITION_SIZE          (0x40000) /* 256 KB for S partition */
-#define FLASH_NS_PARTITION_SIZE         (0x80000) /* 512 KB for NS partition */
+#define FLASH_NS_PARTITION_SIZE         (0xA0000) /* 640 KB for NS partition */
 
 #define FLASH_PARTITION_SIZE            (FLASH_S_PARTITION_SIZE+FLASH_NS_PARTITION_SIZE)
 
